### ISSUE TITLE
OS Installation stalled

### ISSUE THREAD

**Issue by amithkanand** [bloomberg/chef-bach#1049]
While building a VM cluster I found out that OS installation doesn't move forward until you connect to it via serial console to watch the progress.  I am using `Vagrant 2.0.1` with `Virtualbox  5.2.2-119230`.  During my cluster build with 6 VMs I had to connect to each of them using the [virtualbox_serial_console.sh](https://github.com/bloomberg/chef-bach/blob/master/tests/virtualbox_serial_console.sh) script. Also, I got below message when I connected to the VM:
```
$ ./virtualbox_serial_console.sh bach-bcpc-vm4
/home/aanand59/chef-bcpc/virtualbox_env.sh: line 19: bc: command not found
ERROR: VirtualBox 5.2 is less than 4.3.x!
  Only VirtualBox >= 4.3.x is officially supported.
```

**Comment by aespinosa** [comment]
seems like a virtualbox thing. happens in CentOS too https://www.spinics.net/lists/centos-devel/msg15834.html

**Comment by http-418** [comment]
Judging from the mailing list post, we need to manually add a serial port to each vm in `vbox_create.sh`

For the version check, I have a fix in #1209

**Comment by cbaenziger** [comment]
This looks related to #1228. Closing to have a single issue.
